apps/src/netsim/NetSimRouterLogModal.html.ejs

```diff
@@ -21,6 +21,21 @@ var showPacketInfo = headerFields.indexOf(Packet.HeaderType.PACKET_INDEX) > -1 &
  */
 
 /**
+ * @name canToggleRouterLogMode
+ * @type {boolean}
+ */
+
+/**
+ * @name currentTrafficFilter
```

Comment:
This is just an effort to document the locals that this template expects; they're the variables passed in from NetSimRouterLogModal::render().  I feel like ejs has always had this problem of things being loosely coupled, and I don't know if there's a better approach.